docs/src/cli/sample_inverted_index.md

```diff
@@ -5,3 +5,23 @@
 ```
 <!-- cmdrun ../../../build/bin/sample_inverted_index --help -->
 ```
+
+## Description
+Creates a smaller inverted index from an existing one by sampling postings or documents. The purpose of this tool is to reduce time and space requirements while preserving the main statistical properties of the original collection, making it useful for faster experiments and debugging.
```

Comment:
Let's keep the text width to 80 characters. I suggest using some kind of markdown plugin in your editor. For example, in vim you can use `gww` to format the current line or `gwip` to format an entire paragraph (if it's already broken down into lines but you edited some text and want to fix the formatting without rearranging all of it by hand).